benchmarks/persistance/s3_persistance.py

```diff
@@ -0,0 +1,89 @@
+"""Classes for persisting and loading experiment results to a S3-Bucket."""
```

Comment:
"using a S3-Bucket", otherwise it will be hard to find a preposition fitting for both "persisting" and "loading"

Reply:
Currently persisting is added only and I forgot to change the module description. Would rather remove the loading part for now :)

Reply:
Ok, then feel free to adjust and resolve.